### PR TITLE
Remove the AutoCloseable interface in Cli

### PR DESCRIPTION
**Pull request by darionyaphet**
### Description 
Currently, `Cli` have implement `AutoCloseable` and `Closeable` interface but `Closeable` extend `AutoCloseable` so I try to remove duplicate interface .

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

**Comment by darionyaphet** [comment]
[clabot:check]

**Comment by darionyaphet** [comment]
[clabot:check]

File: ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java [hunk 1]
```diff
@@ -73,7 +73,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class Cli implements Closeable, AutoCloseable {
+public class Cli implements Closeable {
```
Comment (apurvam) [inline ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java]: Would it make sense to implement `AutoCloseable` instead? @confluentinc/ksql  WDYT?

Comment (big-andy-coates) [reply ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java]: Don't think it much matters.

**Comment by big-andy-coates** [comment]
Thanks @darionyaphet !

**Comment by darionyaphet** [comment]
thanks ~ @big-andy-coates @hjafarpour @apurvam

**Pull request merged**